pkg/tests/testinfra/testinfra.go

```diff
@@ -195,6 +196,12 @@ func CreateGrafDir(t *testing.T, opts ...GrafanaOpts) (string, string) {
 			_, err = securitySect.NewKey("content_security_policy", "true")
 			require.NoError(t, err)
 		}
+		if len(o.EnableFeatureToggles) > 0 {
```

Comment:
This enables us to do feature toggling within the integration test. cc: @aknuds1 

The looping through options threw me off at the beginning but I guess it's a common idiom for optional params.